api-tests/server/auth_test.go

```diff
@@ -617,3 +620,11 @@ func deleteServiceToken(t *testing.T, serviceAccountID, serviceTokenID int) {
 
 	require.Equalf(t, http.StatusOK, resp.StatusCode, "failed to delete service token, status code: %d, response: %s", resp.StatusCode, b)
 }
+
+func sanitizeSAName(name string) string {
```

Comment:
I wonder if we can't keep and then import this method from `github.com/percona/pmm/managed/utils/strings` package?

Reply:
It is to avoid dependencies between managed and API tests. To keep them separate. We doing this in more places. It was discussed with Nurlan.